app/controllers/v0/vic/profile_photo_attachments_controller.rb

```diff
@@ -26,6 +26,8 @@ def show
         raise Common::Exceptions::RecordNotFound, guid unless GUID_PATTERN.match(guid)
 
         form_attachment = ::VIC::ProfilePhotoAttachment.where(guid: guid).first
+        raise Common::Exceptions::RecordNotFound, guid unless form_attachment
```

Comment:
can you change this to `form_attachment = ::VIC::ProfilePhotoAttachment.find_by!(guid: guid)`? find_by is faster and it will automatically raise the record not found